website_and_docs/content/documentation/grid/getting_started.zh-cn.md

```diff
@@ -214,113 +203,83 @@ driver.quit();
 
 ## Querying Selenium Grid
 
-After starting a Grid, there are mainly two ways of querying its status, through the Grid 
-UI or via an API call.
+启动 Grid 后，主要有两种方式查询其状态，通过 Grid UI 或通过 API 调用。
 
-The Grid UI can be reached by opening your preferred browser and heading to 
-[http://localhost:4444](http://localhost:4444).
+可以通过打开您喜欢的浏览器并前往[http://localhost:4444](http://localhost:4444)。
 
-API calls can be done through the [http://localhost:4444/status](http://localhost:4444/status)
-endpoint or using [GraphQL]({{< ref "advanced_features/graphql_support.md" >}})
+API 调用可以通过 [http://localhost:4444/status](http://localhost:4444/status) 端点或使用 [GraphQL]({{< ref "advanced_features/graphql_support.md" >}})
 
 {{% pageinfo color="primary" %}}
-For simplicity, all command examples shown in this page assume that components are running
-locally. More detailed examples and usages can be found in the
-[Configuring Components]({{< ref "/configuration" >}}) section.
+为简单起见，本页中显示的所有命令示例均假定组件正在运行
```

Comment:
so, are you translating line by line? Seems it should be treating as "running locally" ~~~

Reply:
fixed